code/src/main/webapp/html/faq.html

```diff
@@ -0,0 +1,22 @@
+
+<!DOCTYPE HTML>
+<html>
+<head>
+   <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+   <link rel="stylesheet" href="../CSS/style.css">
+   <script src="../JavaScript/script.js"></script>
+   <title></title>
+</head>
+<body>
+  <div class="general-background">
+    
+  </div>
+  <div class="FAQ" style="text-align:center, display: inline-block">
+      <div id="FAQ-q1" style="text-align:center, display: inline-block">
+        <h1 class="FAQ-questions"> What is Contact Tracing? </h1>
+        <p class="FAQ-answers"> Contact tracing is the process of identification of persons who may have come into contact with an infected person and subsequent collection of further information about these contacts. 
+          <br>TL;DR If you're positive for COVID-19, we help let people you've been in contact with know.<p>
```

Comment:
nit - avoid using the BR tag. Two ways of approaching this:

have faq-answers be a <div> that contains two <p>s, or have two <p>s each of which has an faq-answers class.